src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carPrice = 40;
+  let result = 0;
+
+  if (days >= 7) {
+    result = days * carPrice - 50;
+
+    return result;
```

Comment:
The `return result;` statement is repeated in each conditional block. Consider calculating the result in each block and returning it once at the end of the function to avoid redundancy.